product_docs/docs/epas/14/epas_compat_reference/02_the_sql_language/02_data_types/index.mdx

```diff
@@ -8,27 +8,36 @@ legacyRedirectsGenerated:
 
 <div id="data_types" class="registered_link"></div>
 
-The following table shows the built-in general-purpose data types.
+This table lists all the Oracle data types and provides the following information for each. If:
+- The data type does not apply to Postgres databases.
+- EDB Postgres Advanced Server has implemented an compatible replacement for the Oracle data types that don't have a suitable equivalent in PostgreSQL. The description includes a link to the implementation details in the EDB Postgres Advanced Server documentation.
+- EDB Postgres Advanced Server provides synonyms for PostgreSQL types that can be used directly in place of the equivalent Oracle type.  The description includes a link to the implementation details in the PostgreSQL docs. 
 
```

Comment:
This table lists all the Oracle data types and provides information for each data type. This information includes whether:
- The data type applies to Postgres databases.
- EDB Postgres Advanced Server offers a compatible replacement for the Oracle data types that don't have a suitable equivalent in PostgreSQL. The description includes a link to the implementation details in the EDB Postgres Advanced Server documentation.
- EDB Postgres Advanced Server provides synonyms for PostgreSQL types that you can use in place of the equivalent Oracle type. The description includes a link to the implementation details in the PostgreSQL documentation.